sass/index.scss

```diff
@@ -229,7 +229,60 @@ section {
 
 // Hero
 #hero {
+    display: flex;
+    flex-direction: column;
+
+    #hero-video{
```

Comment:
nit: sPacE beFore bRacE
(it's not a Peter "nit" Kos review without one of these, eh?)